pandas/tests/indexes/multi/test_duplicates.py

```diff
@@ -304,3 +304,17 @@ def test_duplicated_drop_duplicates():
     assert duplicated.dtype == bool
     expected = MultiIndex.from_arrays(([2, 3, 2, 3], [1, 1, 2, 2]))
     tm.assert_index_equal(idx.drop_duplicates(keep=False), expected)
+
+
+@pytest.mark.parametrize(
+    "array,expected",
+    [
+        (
+            [0, 1j, 1j, 1, 1 + 1j, 1 + 2j, 1 + 1j],
+            Series([False, False, True, False, False, False, True], dtype=bool),
+        )
+    ],
+)
+def test_duplicated_series_complex_numbers(array, expected):
+    result = Series(array, dtype=np.complex64).duplicated()
```

Comment:
Is dtype=np.complex64 (instead of np.complex128) intentional? It is probably good idea to test both np.complex128 and np.complex64 (and not only in this test case)-